### PR TITLE
fix se-3883: add redirect for openwebgames.org

### PR DESCRIPTION
**Pull request by lonnen**
JIRA ticket: https://mozilla-hub.atlassian.net/browse/SE-3883

### CODE REVIEW

**Review by ahoneiser — CHANGES_REQUESTED** [review]
lines 1256-1259 need to be dropped, generation of the nginx config will otherwise fail. if openwebgames.com & www.openwebgames.com should both be redirected to the same place, this is unnecessary to keep :slightly_smiling_face:

File: prod-refractr.yml [hunk 1]
```diff
@@ -1256,3 +1252,12 @@ refracts:
 - github.com/mozilla/contribute.json:
   - contributejson.org
   - www.contributejson.org
+
+# bug 1255190
+- www.openwebgames.com/: openwebgames.com
+  status: 302
+
```
Comment (ahoneiser) [inline prod-refractr.yml:1259]: we can get rid of this

Comment (lonnen) [reply prod-refractr.yml]: Thanks! Removed in [`8140f20` (#311)](https://github.com/mozilla-it/refractr/pull/311/commits/8140f20b6cb47b70a6d5ce15eae839e52f1b4626)

**Comment by lonnen** [comment]
🙌

**Pull request merged**